source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,127 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
 B. Define a JSON Schema
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Proin non mi a felis luctus vulputate nec pharetra odio. Ut pretium
-scelerisque nulla, sed euismod nisi pellentesque sit amet. Proin mollis
-mauris eu libero ullamcorper rutrum. Pellentesque vitae enim sed magna
-accumsan maximus et quis mi. Ut vel laoreet turpis. Vivamus vitae odio
-sed arcu finibus pretium vel maximus nisi. Nullam gravida consequat
-porta.
+An important concept in database interaction is `data validation 
+<https://docs.mongodb.com/manual/core/schema-validation/>`_. MongoDB 
+works to make this as simple as possible, using 
+`JSON Schema <https://docs.mongodb.com/manual/reference/operator/query/jsonSchema/#json-schema>`_ 
+for validation during insert and update operations. Within the context of Field Level Encryption,
+JSON Schema is used to specify several configuration details, allowing the drivers
+to automatically encrypt and decrypt fields. 
+
+* To do so, you need to specify two things in JSON Schema for each encrypted field:
+    a) The encryption type (deterministic or random)
+    b) The data key
+    c) The BSON Type
+* You can use a single data key for all encrypted fields or you can use individual data keys that are specific to each field.
+
+
+The following table represents the fields of data, MedcoMD will be using in their Medical Care Management System. Each field
```

Comment:
How's this (it's most similar to option 1 in terms of using the medco example ): "The MedcoMD engineers receive specific requirements for the fields of data. Each field will be either unencrypted or encrypted. Encrypted fields will be encrypted through `Deterministic {link}` Encryption or `Random{link}` encryption. The following table illustrates the data model of the Medco Management System."


I decided not to go with #2 for several reasons:
* if we talk about filterability here we also have to talk about the impact of cardinality of a field in deciding what encryption to use
* the section with the specific field (syntax and explanation), explains why we made specific choices and talks about filterability and cardinality

I decided to not go w/ # 3 because we need to link to ravind's deterministic & random explanation and the table needs some explanation beyond only having a title

Reply:
I also like this new paragraph because it uses an active voice. I've updated the staging link, check it out and let me know what you think @ccho-mongodb

Reply:
I meant a formatted note, e.g.:
```
.. note::
   This helper code is not the full...
```
Also, a nitpick: avoid using "helper code" twice in a row in the same sentence.

Reply:
I see, I fixed it into a formatted note and removed one of the 'helper code' word uses

Reply:
This is really weird -- review comment threads seem to be breaking. My comment "I meant a formatted note, e.g..." has nothing to do with my comment here. I think what is happening is that changes are being committed that have a timestamp before my comment was posted.

Reply:
Are you okay w/ resolving this discussion if you feel your concerns are addressed? Let me know if there's something left to fix w/ this comment thread

@ccho-mongodb